packages/dds/tree/src/simple-tree/discrepancies.ts

```diff
@@ -401,7 +559,7 @@ function* compareMaps<K, V1, V2>(
  * `isRepoSuperset` will determine that a can never be the superset of b. In contrast, `allowsRepoSuperset` will continue
  * validating internal fields.
  */
-export function isRepoSuperset(view: TreeStoredSchema, stored: TreeStoredSchema): boolean {
+export function isRepoSuperset(view: FieldSchema, stored: TreeStoredSchema): boolean {
```

Comment:
given the two arguments don't have the same types, and neither is a TreeStoredSchemaRepository, I think its time to update the name of this function.

Or maybe delete this function? Do we actually have a use case for this anymore? I think our only need for comparing view and stored schema needs more than just a boolean out.